sources/tools/Stride.StorageTool/README.md

```diff
@@ -0,0 +1,22 @@
+# Stride.StorageTool
+
+This is simple Stride3D Asset Bundle previewer.
```

Comment:
The name of the engine is `Stride`. *3D* only exists on the website URL.

Reply:
done!